featurehub_config.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import annotations
+
+from client_context import ClientContext, ClientEvalFeatureContext, ServerEvalFeatureContext
+from edge_service import EdgeService
+from featurehub_repository import FeatureHubRepository
+
+
+class FeatureHubConfig:
+    # readyness: Readyness;
+
+    _api_keys = []
+    _edge_url: str
+    _client_eval: bool
+    _repository: FeatureHubRepository
+
+    def __init__(self, edge_url, *api_key):
+        self._repository = FeatureHubRepository()
+        self._edge_url = edge_url
+
+        if not edge_url or not api_key:
+            raise TypeError('api_key and edge_url must not be null')
+
+        for key in api_key:
+            self._api_keys.append(key)
+
+        self._client_eval = '*' in self._api_keys[0]
```

Comment:
i would check all keys and ensure they are all client side or all server side and fail if there is a mix

Reply:
added on line  23